src/components/ChainVerificationAnimation/BorderAnimation.css

```diff
@@ -1,29 +1,38 @@
+/* CSS Variables */
+:root {
+  --border-width-base: 3px;
+  --border-width-large: 3px;
```

Comment:
What's the point of these two separate variables? Did you mean to make the base 2px?